app/src/main/java/com/example/spork/recommendation/RecommendationScore.java

```diff
@@ -0,0 +1,112 @@
+package com.example.spork.recommendation;
+
+import com.example.spork.Restaurant;
+import com.example.spork.onboarding.OnboardingFragment1;
+import com.example.spork.onboarding.OnboardingFragment2;
+import com.example.spork.onboarding.OnboardingFragment3;
+import com.example.spork.onboarding.OnboardingFragment4;
+
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class RecommendationScore {
+    public static final String TAG = "RecommendationScore";
+    public static final int PRICE_SCALE = 4;
+    public static final int RATING_SCALE = 5;
+    public static final int POPULARITY_SCALE = 5;
+    public static final int PROXIMITY_SCALE = 5;
+
+    private List<Restaurant> restaurantList;
+
+    // assigned arbitrary weights for testing purposes
+    private double priceWeight = 0.4;
+    private double ratingWeight = 0.3;
+    private double popularityWeight = 0.2;
+    private double proximityWeight = 0.1;
```

Comment:
Do you intend to change this in a future PR or modify this further in this PR? I recall you mentioning making a UI for users to select preferences.

Reply:
will implement in a new branch!